src/datakit/main.ml

```
@@ -45,11 +45,26 @@ let make_task msg =
   let date = Int64.of_float (Unix.gettimeofday ()) in
   Irmin.Task.create ~date ~owner:"datakit <datakit@docker.com>" msg
 
+(* FIXME: this is a bit ridiculous *)
+module Contents_string = struct
+  open Irmin.Contents.String
+  type t = string
+  let equal = equal
+  let compare = compare
+  let hash = hash
+  let to_json = to_json
+  let of_json = of_json
+  let size_of = size_of
+  let write = write
+  let read = read
+  let merge _ = merge []
```

I'm assuming we're not using this. Maybe raise an exception if it gets called?